content/en/docs/reference/access-authn-authz/certificate-signing-requests.md

```diff
@@ -524,7 +524,7 @@ Some points to note:
   You can get the content using this command: 
 
   ```shell
-  cat myuser.csr | base64 | tr -d "\n"
+  cat myuser.csr | base64 -w 0
```

Comment:
This assumes that the user has `base64` from GNU coreutils or a clone. We can't make that assumption.

For example, I think macOS doesn't support `-w` (it has `-b` instead).

Reply:
Just checked on macOS and you're right it's `-b`.
Maybe using flags with base64 deserves to be mentioned in addition to the `| tr -d "\n"`, as it's simpler.

Reply:
I'm not convinced; we're teaching Kubernetes more than we're teaching the `base64` tool.